ext/storage_sources/gcp_store/README.md

```diff
@@ -6,56 +6,89 @@ This extension allows WiredTiger storage source extensions to read from and writ
 This section describes how to build WiredTiger with the GCP extension enabled.
 
 ### Requirements
+<li> CMake 3.11 or higher
+<li> G++ 8.4 or higher
 <li> Abseil LTS 20230125
 <li> nlohmann_json library 3.11.2
 <li> crc32c 1.1.2<p>
 
 ### How to install requirements (skip this step if requirements have been met)
-Abseil LTS 20230125
+If the CMake version is not 3.11 or higher update your CMake to 3.11 using the following.
+```bash
+sudo apt remove cmake
+wget https://cmake.org/files/v3.11/cmake-3.11.0.tar.gz
+tar xf cmake-3.11.0.tar.gz
+
+cd cmake-3.11.0
+
+./configure
+make -j $nproc
```

Comment:
On Linux `nproc` is an executable. It doesn't look like nproc is defined prior to this using `export` so I think this should be `$(nproc)` and that is true throughout this document.

Reply:
thanks for picking this up! I'm amended this in https://github.com/wiredtiger/wiredtiger/pull/8803/commits/6500c6fc62b9365f75a883d7f5db2df08aaa305e